### PR TITLE
DCOS-61091 & 61090: empty state components

### PR DESCRIPTION
**Pull request by mperrotti**
## Screenshots
Screen Shot 2019-11-26 at 2 38 38 PM
Screen Shot 2019-11-26 at 2 38 52 PM

### CODE REVIEW

File: packages/emptyState/components/EmptyStateWithGraphic.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const EmptyStateWithGraphic: React.SFC<EmptyStateWithGraphicProps> = ({
+  graphicSrc,
+  graphicDimensions,
+  heading,
+  children,
+  primaryAction,
+  secondaryAction
+}) => {
+  const hasActions = primaryAction || secondaryAction;
+  return (
```
Comment (GeorgiSTodorov) [inline packages/emptyState/components/EmptyStateWithGraphic.tsx]: Can we reuse `EmptyState` here?

Comment (mperrotti) [reply packages/emptyState/components/EmptyStateWithGraphic.tsx]: We could, but why would we?

Comment (GeorgiSTodorov) [reply packages/emptyState/components/EmptyStateWithGraphic.tsx]: For consistency.

Comment (mperrotti) [reply packages/emptyState/components/EmptyStateWithGraphic.tsx]: There is a subtle (but intentional) difference with the spacing between the headline and the body content

Comment (GeorgiSTodorov) [reply packages/emptyState/components/EmptyStateWithGraphic.tsx]: Okay, makes sense.

**Pull request merged**